python/mead/downloader.py

```diff
@@ -0,0 +1,300 @@
+import sys
+
+def get_file(file_name):
```

Comment:
method repeats delete_file().

this method should probably not be deleting anything, given the name.